README.md

```diff
@@ -1 +1,25 @@
-# group-four-BE
\ No newline at end of file
+# group-four-BE
+## How to Make Contributions to this repo
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Above](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7777164)
```suggestion

## How to Make Contributions to this repo
```